python/gen-fixtures.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env sh
+#
+# Copy static files of Python repository into fixtures directory
+#
+set -euo pipefail
+readonly image_name='busybox:latest'
+
+# NOTE: $0 corresponds to the the script's name only in some shells and in some
+# contexts.
+show_help() {
+fmt <<EOF
+usage: gen-fixtures.sh <output_dir> <assets_dir>
+
+Create <output_dir>. Copy Python fixture data from <assets_dir> and place the 
+results into <output_dir>.
+
+EOF
+cat <<EOF
+<output_dir>
+    The directory into which generated fixtures are placed.
+EOF
+}
+
+# Fetch output_dir from user.
+if [ "$#" -lt 2 ]; then
+    echo 1>&2 'Error: Too few arguments received.'
+    echo 1>&2
+    show_help 1>&2
+    exit 1
+elif [ "$#" -gt 2 ]; then
+    echo 1>&2 'Error: Too many arguments received.'
+    echo 1>&2
+    show_help 1>&2
+    exit 1
+else
+    output_dir="$(realpath --canonicalize-missing "${1}")"
+    assets_dir="$(realpath "${2}")"
+fi
+
+# Create output_dir and generate fixture data.
+mkdir "${output_dir}"
+cp -r "${assets_dir}" "${output_dir}"
```

Comment:
This causes the entire "assets" and "test_py" directories to be copied to the `fixtures/python`, producing:

```
fixtures/
└── python
    └── assets
        └── test_py
            ├── packages
            │   └── source
            │       └── s
            │           └── shelf-reader
            │               ├── shelf_reader-0.1-py2-none-any.whl
            │               └── shelf-reader-0.1.tar.gz
            ├── pypi
            │   └── shelf-reader
            │       └── json
            │           └── index.html
            └── simple
                ├── index.html
                └── shelf-reader
                    └── index.html
```

Consider dropping the `test_py` directory and writing this instead:

```sh
cp -rt "${output_dir}" "${assets_dir}"/*
```